src/utilities/functions/index.js

```diff
@@ -19,6 +19,44 @@ export const timeAgo = (dateCreated) => {
   return `${configuredTime} ${showType} ago`;
 };
 
+export const convertTime = (time24) => {
+  let ts = time24;
+  let H = +ts.substr(0, 2);
+  let h = H % 12 || 12;
+  h = h < 10 ? h : h;
+  let ampm = H < 12 ? "AM" : "PM";
+  ts = h + ts.substr(2, 3) + ampm;
+  return ts;
+};
+
+export const formatDate = (date) => {
+  let d = new Date(date),
+    month = "" + (d.getMonth() + 1),
+    day = "" + d.getDate(),
+    year = d.getFullYear();
+
+  if (month.length < 2) month = "0" + month;
+  if (day.length < 2) day = "0" + day;
+
+  return [year, month, day].join("-");
+};
```

Comment:
neat little functions here. There are functions within Date prototype that do I believe the same or similar thing, but honestly, I don't think the code is much shorter. cool stuff.